tests/fixtures/fixtures.h

```diff
@@ -33,9 +33,9 @@ template <typename T, typename RT = typename std::enable_if<std::is_integral_v<T
 std::vector<RT>
 GenerateVectors(uint64_t count,
                 uint32_t dim,
+                int seed = 47,
                 T min = std::numeric_limits<T>::lowest(),
-                T max = std::numeric_limits<T>::max(),
-                int seed = 47) {
+                T max = std::numeric_limits<T>::max()) {
```

Comment:
p.s. The usage of this func in other function is like: GenerateVectors(count, dim, seed). So the old version assign seed to min, trigger a bug.

Reply:
the override `GenerateVectors(uint64_t count, uint32_t dim, bool need_normalize = true, int seed = 47)` also need adjust seed parms ?

Reply:
done